packages/framework/aqueduct/src/container-runtime-factories/containerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -51,18 +50,26 @@ export class ContainerRuntimeFactoryWithDefaultDataStore extends BaseContainerRu
 		defaultFactory: IFluidDataStoreFactory;
 		registryEntries: NamedFluidDataStoreRegistryEntries;
 		dependencyContainer?: IFluidDependencySynthesizer;
-		/** @deprecated Will be removed in future major release. Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md */
+		/** @deprecated Will be removed once Loader LTS version is "2.0.0-internal.7.0.0". Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md */
 		requestHandlers?: RuntimeRequestHandler[];
 		runtimeOptions?: IContainerRuntimeOptions;
 		provideEntryPoint?: (runtime: IContainerRuntime) => Promise<FluidObject>;
 	}) {
 		const requestHandlers = props.requestHandlers ?? [];
 		const provideEntryPoint = props.provideEntryPoint ?? getDefaultFluidObject;
 
+		const getDefaultObject = async (
+			req: IRequest,
+			runtime: IContainerRuntime,
+		): Promise<IResponse> => {
+			return req.url === "" || req.url === "/" || req.url.startsWith("/?")
+				? { mimeType: "fluid/object", status: 200, value: await provideEntryPoint(runtime) }
+				: create404Response(req);
```

Comment:
Here and anywhere else where this handler will be composed using RuntimeRequestHandlerBuilder - if it returns a 404 response, it prevents subsequent requestHandlers from activating (RuntimeRequestHandlerBuilder expects a return value of `undefined` to permit falling through to the next handler).  So probably want this to return `undefined` instead of 404?